plugins/in_tail/tail.c

```diff
@@ -611,6 +611,14 @@ static struct flb_config_map config_map[] = {
      FLB_CONFIG_MAP_INT, "progress_check_interval_nsec", "0",
      0, FLB_TRUE, offsetof(struct flb_tail_config, progress_check_interval_nsec),
     },
+    {
+     FLB_CONFIG_MAP_TIME, "read_interval_sec", "0s",
```

Comment:
Since this is only for the stat(2) based backend, I would suggest to add a prefix so users are not confused, e.g:

```
stat.read_interval_sec
```

and 

```
stat.read_interval_nsec
```